apps/api-journeys/prisma/factories/nua2.ts

```diff
@@ -0,0 +1,583 @@
+      blockType: 'StepBlock',
+      extraAttrs: {
+        locked: false,
+        nextBlockId
+      },
+      parentOrder: 4
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'VideoTriggerBlock',
+      parentBlockId: video1.id,
+      extraAttrs: {
+        triggerStart: 272,
+        action: {
+          gtmEventName: 'trigger',
+          blockId: step4.id
+        }
+      }
+    }
+  })
+  const image2Id = uuidv4()
+  const card4 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'CardBlock',
+      parentBlockId: step4.id,
+      extraAttrs: {
+        themeMode: ThemeMode.dark,
+        themeName: ThemeName.base,
+        coverBlockId: image2Id
+      },
+      parentOrder: 0
+    }
+  })
+
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: card4.id,
+      extraAttrs: {
+        content: 'A QUOTE',
+        variant: 'h6',
+        color: 'primary',
+        align: 'left'
+      },
+      parentOrder: 1
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: card4.id,
+      extraAttrs: {
+        content:
+          "...one of the soldiers pierced Jesus' side with a spear, bringing a sudden flow of blood and water.",
+        variant: 'subtitle1',
+        color: 'primary',
+        align: 'left'
+      },
+      parentOrder: 2
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: card4.id,
+      extraAttrs: {
+        content: '- The Bible, John 19:34',
+        variant: 'caption',
+        color: 'primary',
+        align: 'left'
+      },
+      parentOrder: 3
+    }
+  })
+  await prisma.block.create({
+    data: {
+      id: image2Id,
+      journeyId: journey.id,
+      blockType: 'ImageBlock',
+      parentBlockId: card4.id,
+      extraAttrs: {
+        src: 'https://images.unsplash.com/photo-1616977545092-f4a423c3f22e?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=765&q=80',
+        alt: 'quote',
+        width: 1920,
+        height: 1080,
+        blurhash: 'L9Db$mOt008_}?oz58M{.8o#rqIU'
+      },
+      parentOrder: 0
+    }
+  })
+
+  //   sixth step is now a video
+  const step5 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'StepBlock',
+      extraAttrs: {
+        locked: false,
+        nextBlockId
+      },
+      parentOrder: 5
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: card4.id,
+      extraAttrs: {
+        label: 'What does it mean?',
+        variant: 'contained',
+        color: 'primary',
+        size: 'medium',
+        startIcon: {
+          name: 'PlayArrow'
```

Comment:
```suggestion
          name: 'ContactSupport'
```
When we have it added